src/promptflow-tools/promptflow/tools/yamls/open_source_llm_with_custom_llm.yaml

```diff
@@ -0,0 +1,25 @@
+promptflow_openllm.tool.open_source_llm.OpenSourceLLM.call_with_custom_llm:
```

Comment:
This does not match the method in your source code. Please kindly share screenshots of tool verification with us, which may make it easier for us to review.